realscale.inx

```diff
@@ -16,16 +17,52 @@
     <item value="m">m</item>
     <item value="km">km</item>
   </param>
+  <param name="desc_unit" type="description">Large objects need a scale to be drawn on the page:</param>
+  <param name="showscale" type="boolean" _gui-text="Show Scale Ruler?">true</param>
+  <param name="drawnscale" type="enum" _gui-text="Drawing Scale:                    " _gui-description="-world unit for drawing">
```

Comment:
The only reason is to meet the established standards. E.g. if you're doing this and you use 23.71 as a scale - you won't find any rulers that you can measure off the drawing and read the direct amount. Which you can with any of these. As I indicated in the initial issue - there are architectural, metric, and engineering standards for a set of expected and useful scales.
So I will probably break this into three sections then. imperial, metric, and manual and have user choose...